assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -458,6 +458,15 @@ che.infra.kubernetes.pvc.jobs.memorylimit=250Mi
 # https://docs.openshift.com/container-platform/latest/architecture/additional_concepts/storage.html#pv-access-modes
 che.infra.kubernetes.pvc.access_mode=ReadWriteOnce
 
+# Defines whether Che Server should wait workspaces PVCs to become bound after creating.
+# It's used by all PVC strategies.
+#
+# It should be set to `false` in case if `volumeBindingMode` is configured to `WaitForFirstConsumer`
+# otherwise workspace starts will hangs up on phase of waiting PVCs.
+#
+# Default value is true(means that PVCs should be waited to be bound)
+che.infra.kubernetes.pvc.wait_bound=true
```

Comment:
that is making us backward compatible with OC3 by default but not with OC4. Am I right?
For some reasone, I think we need to make it opposite. 
@sleshchenko @l0rd wdyt?